README.md

```diff
@@ -258,21 +258,590 @@ As part of the deployment, you'll need to set up [git integration](https://rasa.
+make aws-eks-cluster-delete AWS_EKS_CLUSTER_NAME=financial-demo-production
+```
+
+### Configure `kubeconfig`
+
+Add the EKS  cluster information to `~/.kube/config`, and set the `current-context` to that cluster:
+
+```bash
+make aws-eks-cluster-update-kubeconfig AWS_EKS_CLUSTER_NAME=financial-demo-production
+
+# make sure kubectl is now looking at the correct EKS cluster
+make kubectl-config-current-context
+```
+
+### Install/Upgrade Rasa Enterprise
+
+#### Build & push action server docker image
+
+Build, run, test & push the action docker server image, with name:
+
+- `<ECR URI>/financial-demo:<current branch name>`
+
+```bash
+# Verify that the correct rasa-sdk is selected in `Dockerfile`
+# ==> FROM rasa/rasa-sdk:....
+#
+# Then, build the image with:
+make docker-build
+
+make docker-run
+make docker-test
+make docker-stop
+
+# Login & push the docker image to the AWS ECR repository
+make aws-ecr-docker-login
+make docker-push
+```
+
+#### Install/Upgrade Rasa Enterprise
+
+Install/Upgrade Rasa Enterprise, using the action server docker image that was uploaded to the ECR repository:
+
+```bash
+# make sure kubectl is looking at the correct EKS cluster
+# -> if not, configure kubeconfig as described above
+make kubectl-config-current-context  AWS_EKS_CLUSTER_NAME=financial-demo-production
+
+# create namespace `my-namespace`
+make aws-eks-namespace-create
+
+# create/refresh gcr-pull-secret, for Rasa Enterprise image
+make pull-secret-gcr-create
+
+# create/refresh ecr-pull-secret, for action server image
+make pull-secret-ecr-create
+
+# Verify that:
+#
+# the correct Rasa Enterprise version is selected in `Makefile`
+# ==> RASAX_TAG := ......
+# 
+# the correct rasa version is selected in `requirements.txt`
+# ==> rasa[spacy]==....
+#
+# Install/Upgrade Rasa Enterprise with the action server
+make rasa-enterprise-install
+
+# Check Rasa Enterprise Health
+make rasa-enterprise-check-health
+
+# To troubleshoot, highly recommended to use OCTANT, see Appendix C
+```
+
+### Train, test & upload model to S3
+
+```bash
+# Train the model: `models/<current branch>.tar.gz`
+make rasa-train
+
+# In another window, start duckling server
+docker run -p 8000:8000 rasa/duckling
+
+# Run the end-2-end tests
+make rasa-test
+
+# Upload `models/<current branch>.tar.gz` to S3
+# Note: This does not mean the model is deployed to Rasa Enterprise,
+#       which is done in the next step.
+make aws-s3-upload-rasa-model
+```
+
+### Deploy, Tag & Smoketest the trained model
+
+```bash
+# Configure ~/.kube/config and set current_context
+make aws-eks-cluster-update-kubeconfig  AWS_EKS_CLUSTER_NAME=financial-demo-production
+
+# Deploy rasa model
+make aws-s3-download-rasa-model
+make rasa-enterprise-model-delete
+make rasa-enterprise-model-upload
+make rasa-enterprise-model-tag
+
+# Wait about 1 minute, so rasa-production can download, upack & load the model
+# Smoketest
+make rasa-enterprise-smoketest
+```
+
+### DNS
```

Comment:
This step is optional right?

Reply:
Correct. 
I explained that it is optional and what it is for.